src/Sql/Sql/Auditing/Cmdlet/SetAzSqlDatabaseAudit.cs

```diff
@@ -109,6 +109,12 @@ public class SetAzSqlDatabaseAudit : SqlDatabaseAuditCmdlet
             HelpMessage = AuditingHelpMessages.PassThruHelpMessage)]
         public SwitchParameter PassThru { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = AuditingHelpMessages.UseIdentityMessage)]
+        [ValidateNotNullOrEmpty]
+        public string UseIdentity { get; set; }
```

Comment:
Normally we use SwitchParameter or bool parameter for those takes only two options - true or false, as it's more user friendly. Any particular reason to implement it as a string?
To be specific, in `Set-` cmdlets, we often use an optional Boolean parameter to represent such a choice - true is true, false is false, null is do not update.

Reply:
currently it is behaving like that only and case insensitive. In future If azure wants provides customer the options to choose between user managed identity and system managed identity...I just need to add an extra enum value. If we make as bool, I need to do some breaking changes or might need another parameter.

Reply:
In that case, I recommend following the common pattern in Azure PowerShell to handle managed identity, which is by exposing a `-IdentityType <string>` parameter which accepts `"SystemAssigned", "UserAssigned", "SystemAssigned, UserAssigned", "None"`, and an additional `-IdentityId <string[]>` that accepts user assigned identity IDs.
[New-AzServiceBusNamespace](https://learn.microsoft.com/en-us/powershell/module/az.servicebus/new-azservicebusnamespace?view=azps-9.2.0) is an example.
For now you only need to support "SystemAssigned" and "None", and you don't need `-IdentityId` parameter.

Reply:
This Name can be little confusing for customer for now. As, Managed Identity is supported for auditing using powershell when storage account is behind firewall, it takes the primary identity whatever customer configured. 
Now, when the storage is not behind firewall, Cx don't have option to choose between sas key and identity.
that's why I used `UseIdentity` column. 
I choosed boolean first, But while giving boolean input...I had to provide 1 and 0 in powershell, but true/false is more readable. So, I chose Enum type.

Reply:
OK. Let's keep it this way. Please address the other comments.